src/components/MultiFlatmapVuer.vue

```diff
@@ -377,27 +463,51 @@ export default {
         })
       }
     },
+    /**
+     * TODO: This function is not in use.
+     * Maybe duplicate of ``FlatmapSelected``.
+     * Function to emit ``resource-selected`` event with provided ``resource``.
+     * @arg action
+     */
     resourceSelected: function (action) {
```

Comment:
Can you replace FlatmapSelected with resourceSelected in this file please?

Reply:
Updated - 40aab785f0908224539d25e75b5e54c35c70ba00